<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EndWindow</class>
 <widget class="QMainWindow" name="EndWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>881</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_vol_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QFrame" name="map_frame_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>691</width>
        <height>311</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_m_map_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>671</width>
         <height>291</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/images/images/map.png</pixmap>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorMoscow_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorVolga_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorCaspian_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorNovosibirsk_3">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>160</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="indicatorEnd_3">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>190</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_9">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>10</y>
        <width>161</width>
        <height>511</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLCDNumber" name="lcdNumber_ir_odo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLCDNumber{color:rgb(230, 97, 0);}</string>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Mode::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>999.000000000000000</double>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber_ir_day">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLCDNumber{color:rgb(230, 97, 0);}</string>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Mode::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>999.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="button_ir_end">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>141</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>End</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Odometer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>440</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>KM</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_ir_crew">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Crew</string>
       </property>
      </widget>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_ir">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>330</y>
        <width>691</width>
        <height>192</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'IBM Plex Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
